src/balloon/balloon.jsx

```diff
@@ -335,12 +336,22 @@ export default class Balloon extends React.Component {
         triggerProps['aria-describedby'] = this._contentId;
         triggerProps.tabIndex = '0';
 
-        const newTrigger = React.cloneElement(trigger, triggerProps);
+        const ariaTrigger = this._contentId
+            ? React.cloneElement(trigger, triggerProps)
+            : trigger;
+
+        const newTrigger = getDisabledCompatibleTrigger(
+            React.isValidElement(ariaTrigger) ? (
+                ariaTrigger
+            ) : (
+                <span>{ariaTrigger}</span>
```

Comment:
I suggest adding a qa-hook for the unit tests

Reply:
I have writed the test case in `test/balloon/balloon-spec.js` line 147, is that enough?And what's the qa-hook...?

Reply:
qa-hooks are a way to avoid very general dom searching or searching by existing style related tags that are subject to change. Its not that important, but can prevent unexpected unit test breaks. I don't see us using anywhere else, so to be consistent you can just ignore my comment.